static/js/browser2/statvar_group_node.tsx

```diff
@@ -0,0 +1,208 @@
+                          />
+                        </div>
+                      );
+                    })
+                  : null}
+              </div>
+            ) : null}
+            <div className="svg-node-child">
+              {this.props.data[this.props.statVarGroupId].childStatVarGroups
+                ? this.props.data[
+                    this.props.statVarGroupId
+                  ].childStatVarGroups.map((childStatVarGroup) => {
+                    if (
+                      _.isEmpty(this.props.pathToSelection) ||
+                      this.props.pathToSelection[0] === childStatVarGroup.id
+                    ) {
+                      return (
+                        <div
+                          key={childStatVarGroup.id}
+                          ref={
+                            this.props.pathToSelection.length === 1
+                              ? this.highlightedStatVar
+                              : null
+                          }
+                        >
+                          <StatVarGroupNode
+                            dcid={this.props.dcid}
+                            placeName={this.props.placeName}
+                            statVarGroupId={childStatVarGroup.id}
+                            data={this.props.data}
+                            pathToSelection={this.props.pathToSelection.slice(
+                              1
+                            )}
+                            specializedEntity={
+                              childStatVarGroup.specializedEntity
+                            }
+                            isOpened={
+                              this.props.pathToSelection[0] ===
+                              childStatVarGroup.id
+                            }
+                            isSelected={this.props.pathToSelection.length === 1}
+                          />
+                        </div>
+                      );
+                    }
+                  })
+                : null}
+            </div>
+          </>
+        ) : null}
+      </Collapsible>
+    );
+  }
+
+  private scrollToHighlighted(): void {
+    clearTimeout(this.delayTimer);
+    this.delayTimer = setTimeout(() => {
+      if (this.highlightedStatVar.current !== null) {
+        const rect = this.highlightedStatVar.current.getBoundingClientRect();
+        if (rect.top < 0 || rect.top > window.innerHeight) {
+          this.highlightedStatVar.current.scrollIntoView({
+            behavior: "smooth",
+          });
+        }
+      }
+    }, 400);
+  }
+}
+
+interface statVarHierarchyNodeHeaderPropType {
+  title: string;
+  opened: boolean;
+  highlighted: boolean;
+  nodeType: StatVarHierarchyNodeType;
+}
+
+export class StatVarHierarchyNodeHeader extends React.Component<
+  statVarHierarchyNodeHeaderPropType
+> {
+  render(): JSX.Element {
+    let prefixHtml =
+      this.props.nodeType === StatVarHierarchyNodeType.STAT_VAR ? (
+        <span>&#8226;</span>
```

Comment:
Can you make "&#8226" (and others) as constant with meaningful names.

Reply:
done